.eslintrc.js

```diff
@@ -69,6 +69,10 @@ module.exports = {
     },
   },
   rules: {
+    "@typescript-eslint/no-unused-vars": [
+      errorIfStrict,
+      { vars: "all", args: "after-used", ignoreRestSiblings: true },
+    ],
```

Comment:
what's dis?

Reply:
This was already here i just moved it. So the plugins are in alphabetical order